maths/fermats_little_theorem.dart

```diff
@@ -21,12 +19,22 @@ void main() {
   // a prime number
   int p = 701;
 
-  double a = 1000000000;
+  int a = 1000000000;
   int b = 10;
 
   // using binary exponentiation function, O(log(p)):
-  print((a / b) % p == (a * binary_exponentiation(b, p - 2, p)) % p);
```

Comment:
Why remove this?  When the visitor runs this program then nothing visible will happen.  We are writing these programs primarily for readers to understand and secondarily for computers to test.